.github/ISSUE_TEMPLATE/1-Bug_report.md

```diff
@@ -0,0 +1,45 @@
+---
+name: Bug report
+about: You found a bug in expresslrs. 🐞
+labels: 'bug'
+---
+
+<!-- Please use the following issue template or your issue will be closed -->
+
+## Expected Behavior
+
+<!--- What should have happened? -->
+
+## Current Behavior
+
+<!--- What went wrong? -->
+
+## Steps to Reproduce
+
+<!-- Add relevant code and/or a live example -->
+<!-- Add stack traces -->
+
+1.
+
+2.
+
+3.
+
+4.
+
+## Possible Solution (Not obligatory)
+
+<!--- Suggest a reason for the bug or how to fix it. -->
+
+## Context
```

Comment:
Instead of `Context` I might call it `Additional details you think might be relevant`. Doesn't always have to be as short as possible.

Reply:
Called it "Details".